types/src/transaction/mod.rs

```diff
@@ -635,6 +635,27 @@ impl From<VMStatus> for TransactionStatus {
     }
 }
 
+/// The result of running the transaction through the VM validator.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct VMValidatorResult {
+    status: Option<VMStatus>,
+    gas_price: u64,
```

Comment:
Should we call this something like `normalized_gas_price` or `priority` just to emphasize that it is (or will be) different from the gas price on the `RawTransaction`?